src/abstractdataframe/join.jl

```diff
@@ -182,23 +195,21 @@ function update_row_maps!(left_table::AbstractDataFrame,
 end
 
 """
-Join two DataFrames
+    join(df1, df2; on = Symbol[], kind = :inner)
 
-```julia
-join(df1::AbstractDataFrame,
-     df2::AbstractDataFrame;
-     on::Union{Symbol, Vector{Symbol}} = Symbol[],
-     kind::Symbol = :inner)
-```
+Join two DataFrames
 
 ### Arguments
 
 * `df1`, `df2` : the two AbstractDataFrames to be joined
 
 ### Keyword Arguments
 
-* `on` : a Symbol or Vector{Symbol}, the column(s) used as keys when
-  joining; required argument except for `kind = :cross`
+* `on` : A column, or vector of columns to join df1 and df2 on; If the column(s)
```

Comment:
Use period rather than semicolon. Also add backticks around `(left, right)` and `(left => right)`, and "Tuple" should either be in backticks and in singular, or be in lowercase.

Could you also add an example?